html/js/tree/ISet.js

```diff
@@ -0,0 +1,151 @@
+GTE.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new ISet.
+    * @class
+    * @param {ISet} [parent] Parent ISet. If null, this is root.
+    * @param  {Number} [numOfNodes] Number of nodes of this new information set. If null 1
```

Comment:
The brackets are used when the params are optional. In this case, it is true parent is not optional, but numOfNodes is :+1: